docs/plugins/repository-s3.asciidoc

```diff
@@ -453,6 +453,33 @@ bucket, in this example, named "foo".
 The bucket needs to exist to register a repository for snapshots. If you did not
 create the bucket then the repository registration will fail.
 
+===== Cleaning up multi-part uploads
+
+{es} uses S3's multi-part upload process to upload larger blobs to the
+repository. The multi-part upload process works by dividing each blob into
+smaller parts, uploading each part independently, and then completing the
+upload in a separate step. This reduces the amount of data that {es} must
+re-send if an upload fails: {es} only needs to re-send the part that failed
+rather than starting from the beginning of the whole blob. The storage for each
+part is charged independently starting from the time at which the part was
+uploaded.
+
+If a multi-part upload cannot be completed then it must be aborted in order to
+delete any parts that were successfully uploaded, preventing further storage
+charges from accumulating. {es} will automatically abort a multi-part upload on
+failure, but sometimes the abort request itself fails. For instance, if the
+repository becomes inaccessible or the instance on which {es} is running is
```

Comment:
NIT: maybe replace `instance` with `node` here, it reads funny with the `For instance,` at the beginning of the sentence doesn't it? :)

Reply:
Good point. I replaced `For instance` with `For example` instead, I'd rather not call it a node since that's the name for the ES process itself.